### PR TITLE
Resolved expansion of ! in the shell

### PR DESCRIPTION
**Pull request by jexp**

### CODE REVIEW

File: community/shell/src/main/java/org/neo4j/shell/impl/JLineConsole.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ static JLineConsole newConsoleOrNullIfNotFound( ShellClient client )
 				Class.forName( "jline.console.ConsoleReader" ).newInstance();
 			consoleReader.getClass().getMethod( "setBellEnabled",
 				Boolean.TYPE ).invoke( consoleReader, false );
-			consoleReader.getClass().getMethod( "setBellEnabled", Boolean.TYPE ).invoke(
+			consoleReader.getClass().getMethod( "setExpandEvents", Boolean.TYPE ).invoke(
```
Comment (lutovich) [inline community/shell/src/main/java/org/neo4j/shell/impl/JLineConsole.java:43]: With this setup `bellEnabled` would not be set to `false`. Is this ok? Maybe setting `expandEvents` to `false` should be a new separate statement?

Comment (jexp) [inline community/shell/src/main/java/org/neo4j/shell/impl/JLineConsole.java:43]: if you check the original code, you see that setBellEnabled was in there twice. That's why the change shows up as changing the 2nd repetition of setBellEnabled.
